src/drive/mobile/containers/StackErrorBoundary.jsx

```diff
@@ -0,0 +1,95 @@
+/* global __TARGET__ */
+import React, { Component } from 'react'
+import ReactMarkdown from 'react-markdown'
+import tosIcon from '../assets/icons/icon-tos.svg'
+import { Modal, Icon, Button, translate } from 'cozy-ui/react'
+import styles from '../styles/tosupdated'
+
+const TosUpdatedModal = translate()(({ t, newTosLink, onAccept, onRefuse }) => (
+  <Modal closable={false}>
+    <Modal.ModalHeader />
+    <Modal.ModalDescription className={styles['tosupdated']}>
+      <Icon icon={tosIcon} width={96} height={96} />
+      <h2 className={styles['tosupdated-title']}>{t('TOS.updated.title')}</h2>
+      <ReactMarkdown
+        className={styles['tosupdated-desc']}
+        source={t('TOS.updated.detail', { link: newTosLink })}
+      />
+      <Button
+        extension="full"
+        label={t('TOS.updated.cta')}
+        onClick={onAccept}
+      />
+      <Button
+        subtle
+        size="small"
+        extension="full"
+        label={t('TOS.updated.disconnect')}
+        onClick={onRefuse}
+      />
+    </Modal.ModalDescription>
+  </Modal>
+))
+
+class StackErrorBoundary extends Component {
```

Comment:
What do you think about renaming this component? In the end, it's not acting as a boundary at all, so maybe the name is misleading.